.travis.yml

```diff
@@ -1,4 +1,7 @@
 sudo: required
+jdk:
+  - openjdk8
+  - openjdk11
 language: scala
 scala:
    - 2.12.2
```

Comment:
our scala version is `2.12.7`

Reply:
Good catch. Should we use `TRAVIS_SCALA_VERSION`?